java/code/src/com/redhat/rhn/manager/contentmgmt/ContentManager.java

```diff
@@ -455,6 +455,12 @@ public ContentFilter createFilter(String name, ContentFilter.Rule rule, ContentF
         lookupFilterByNameAndOrg(name, user).ifPresent(cp -> {
             throw new EntityExistsException(cp);
         });
+
+        if (ContentFilter.EntityType.MODULE.equals(entityType)) {
+            // Rules are not applicable for module filters
+            // Set the rule to ALLOW for all module filters to avoid confusion when listing with XMLRPC
+            rule = ContentFilter.Rule.ALLOW;
```

Comment:
I'd prefer, if we slap the user on the hand (e.g. with `IllegalArgumentsException`), propagate it to the XMLRPC API and there re-throw some fault. This way, they will really know that they did something forbidden. If you think this is a bad idea, let me know and we can talk about it :)

Reply:
Done. Now it enforces `ALLOW` consistently everywhere.